worldedit-core/src/main/java/com/fastasyncworldedit/core/history/changeset/AbstractChangeSet.java

```diff
@@ -384,12 +384,12 @@ public Future<?> addWriteTask(final Runnable writeTask, final boolean completeNo
         }
     }
 
-    private void triggerWorker() {
+    private synchronized void triggerWorker() {
```

Comment:
What's the intention here?

Reply:
Eh, tbf there's no real need for it. It's just rarely create an extra task that'll complete quickly I suppose